README.md

```diff
@@ -1,7 +1,7 @@
 # WICG administration repository
 This repo is for administration of the community group.
  
- * [Home page - wicg.io](https://wicg.io) - includes links to active specs. 
+ * [Home page - wicg.io](https://wicg.io) - includes links to active proposals. 
```

Comment:
The link is actually to the proposal's repo [homepage](https://docs.github.com/en/rest/reference/repos#create-an-organization-repository--parameters). In most cases, that's a spec, although some may point at an explainer if that's all they got.